docs/setting-up-ado-pipelines.md

```diff
@@ -0,0 +1,2 @@
+# Adding SBOM generation to your Azure DevOps Pipeline
```

Comment:
Are you going to add this in this PR? If no, maybe remove this file until doc is not ready?

Reply:
Sending the file now